src/scss/components/events-highlights.scss

```diff
@@ -0,0 +1,83 @@
+.event {
+  $root: &;
+  @extend .container;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 80px;
+
+  &.feature {
```

Comment:
Why `.feature` is not under the BEM naming convention? Is it a class that you are going to reuse in other components? If not, please use BEM to name this class.

Reply:
this class is used to differentiate the special events component with the event list component since this same js file will be used for both components

Reply:
In that case, it is better to use the class as a modifier using BEM notation

Reply:
change .feature to .event__feature